code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -141,10 +141,9 @@
 		E.Goto(owner, E.move_to_delay, E.minimum_distance)
 		owner.alpha = 0
 		owner.visible_message("<span class='danger'>[owner] is hit by [attack_text] in the chest!</span>") //We pretend to be hit, since blocking it would stop the message otherwise
-		spawn(40)
-			owner.alpha = initial(owner.alpha)
+		addtimer(VARSET_CALLBACK(owner, alpha, initial(alpha)), 40)
```

Comment:
Does this not set the owners alpha to the initial alpha of the reactive armour?

Reply:
No, since the first argument is `owner`, not `src`, so this modifies `owner`'s alpha.

Reply:
Also actually tested it, works as intended.

Reply:
![image](https://user-images.githubusercontent.com/26465327/128236768-ee5f529b-93d9-4590-8ad3-7c8b57721c20.png)

The var_value is the value inputted, not the value of a variable on datum. Var is a reference to a variable on datum, while var is not.
datum.var = var_value

While this likey won't affect anything anything since reactive armour will always start as 255 and owner will probably be 255, should be noted to look out for other cases of this since this is setting owner.alpha to inital({src.}alpha) rather than initial(owner.alpha)

Reply:
So what should I even do?

Reply:
addtimer(VARSET_CALLBACK(owner, alpha, initial(owner.alpha)), 40)